frontend/packages/console-app/src/components/cloud-shell/CloudshellConfirmationModal.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { confirmModal } from '@console/internal/components/modals/confirm-modal';
+
+const cloudshellConfirmationModal = (action) => {
+  return confirmModal({
+    title: 'Close Terminal?',
+    message: (
+      <>
+        This will close the terminal session. Content in the terminal will not be restored on next
+        session.
+      </>
+    ),
```

Comment:
```suggestion
    message: 'This will close the terminal session. Content in the terminal will not be restored on next session.',
```

With this change, remove the `React` import and rename file to use `.ts` suffix.

Reply:
updated